### PR TITLE
feat(web): add antd icons as atom

### PR DESCRIPTION
**Pull request by issmail-basel**

### CODE REVIEW

File: web/src/components/atoms/Icon/index.tsx [hunk 1]
```diff
@@ -1,6 +1,91 @@
-import { HomeOutlined, DownloadOutlined } from "@ant-design/icons";
+import styled from "@emotion/styled";
```
Comment (KaWaite) [inline web/src/components/atoms/Icon/index.tsx:1]: Is there any reason at the moment you made this component to be useable with more than just the antd icons?
Do we have icons we are using that aren't antd that we want to support? Or are you just keeping this as close or the same to Re:Earth as possible?

File: web/src/components/atoms/Icon/index.tsx [hunk 2]
```diff
@@ -1,6 +1,91 @@
...
+        color={color}
+        style={style}
+        alt={alt}
+        size={sizeStr}
+        onClick={onClick}
+      />
+    );
+  }
+
+  return React.createElement(src); // this is of type (React.ForwardRefExoticComponent)
```
Comment (KaWaite) [inline web/src/components/atoms/Icon/index.tsx:63]: Does this still allow for antd Icon's props? (Like spin, rotate, etc)?
Or does @lavalse design not really make use of those at all?

Comment (issmail-basel) [reply web/src/components/atoms/Icon/index.tsx:63]: @KaWaite , for now, there is no use for antd properties, but can you help me with code, and find a way to use and props with it?

Comment (issmail-basel) [reply web/src/components/atoms/Icon/index.tsx:63]: @KaWaite , can be added when needed, for now, there is no usage for these props.
For now, there is no need (to keep components as generic as possible and to have a way to replace antd icons)
in the future if we need any prop we can simply:
`React.createElement(src, {propName: propValue})`

File: web/src/components/atoms/Icon/index.tsx (hunk 1 above)
Comment (issmail-basel) [reply web/src/components/atoms/Icon/index.tsx:1]: @KaWaite , yes, to have a generic component, where we can use multiple icons formats (to keep components as generic as possible and to have a way to replace antd icons)

**Review by KaWaite — APPROVED** [review]

**Pull request merged**